.eslintrc.js

```diff
@@ -12,5 +17,8 @@ module.exports = {
       },
     },
   ],
-  rules: { "import/order": "off", "react-hooks/exhaustive-deps": "error" },
+  rules: {
+    "import/order": "off",
+    "react-hooks/exhaustive-deps": "warn",
```

Comment:
why do we not want this to be an error?

Reply:
That's a good catch, would probably want error instead since a missing dependency could break the extension!